examples/model_selection/plot_permutation_tests_for_classification.py

```diff
@@ -41,27 +41,28 @@
 # ----------------------
 #
 # Next, we calculate the
-# :func:`~sklearn.model_selection.permutation_test_score` using the original
-# iris dataset, which strongly predict the labels and
-# the randomly generated features and iris labels, which should have
-# no dependency between features and labels. We use the
+# :func:`~sklearn.model_selection.permutation_test_score` for both, the original
+# iris dataset (where features strongly predict the labels) and
```

Comment:
```suggestion
# iris dataset (where there's a strong relationship between features and labels) and
```
Optional suggestion, it's the same wording as we use in the user guide and I feel like this reads clearer?

Reply:
Or 'strong dependency between features and labels' ?